Packs/TrendMicroVisionOne/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Trend Micro Vision One",
     "description": "Trend Micro Vision One is a purpose-built threat defense platform that provides added value and new benefits beyond XDR solutions, allowing you to see more and respond faster. Providing deep and broad extended detection and response(XDR) capabilities that collect and automatically correlate data across multiple security layers—email, endpoints, servers, cloud workloads, and networks—Trend Micro Vision One prevents the majority of attacks with automated protection.",
     "support": "partner",
-    "currentVersion": "4.1.1",
+    "currentVersion": "4.2.0",
```

```suggestion
    "currentVersion": "4.1.2",
```
Please update the release notes file name accordingly